### PR TITLE
Bump to v1.1.1

### PR DESCRIPTION
**Pull request by ocefpaf**
Implementing @dkillick changes from https://github.com/SciTools/conda-recipes-scitools/pull/171 in conda-forge.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,5 +34,6 @@ about:
 
 extra:
     recipe-maintainers:
+        - dkillick
```
Comment (ocefpaf) [inline recipe/meta.yaml:38]: @dkillick is it OK to list you here?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -32,5 +34,7 @@ about:
 
 extra:
     recipe-maintainers:
-        - pelson
+        - bjlittle
+        - dkillick
```
Comment (ocefpaf) [inline recipe/meta.yaml:38]: Ping @bjlittle and @dkillick. Is it OK to list you both as maintainers of `cf_units` in `conda-forge`?

Comment (bjlittle) [inline recipe/meta.yaml:38]: @ocefpaf Great, thanks!

**Comment by ocefpaf** [comment]
The failure in AppVeyor is unrelated to the PR. It is due to the latest conda-build and a fix is already under way.

**Pull request merged**